vllm/model_executor/layers/attention.py

```diff
@@ -355,7 +355,14 @@ def forward(
 
         # Apply rotary embedding to the query and key before passing them
         # to the attention op.
-        query, key = self.rotary_emb(positions, query, key)
+        if not isinstance(self.rotary_emb,
+                          DynamicNTKScalingRotaryEmbeddingQwen):
+            query, key = self.rotary_emb(positions, query, key)
+        else:
+            # need to pass in prompt_token_length to compute dynamic_ntk for qwen
+            query, key = self.rotary_emb(
+                positions, query, key,
+                input_metadata.seq_groups[0][1].prompt_token_length)
```

Comment:
changed 
query, key = self.rotary_emb(
                positions, query, key,
                input_metadata.seq_groups[0][1].prompt_token_length)
to
prompt_token_length = max([seq[1].prompt_token_length for seq in input_metadata.seq_groups ])
query, key = self.rotary_emb(
                positions, query, key,
                prompt_token_length)
can solve this problem